# msg descriptor to publish the blocks positions

string name
float32[] position
float32[] rotation
